spec/models/work_package/work_package_planning_spec.rb

```diff
@@ -170,36 +170,25 @@
         expect(planning_element).not_to be_valid
 
         expect(planning_element.errors[:project]).to be_present
-        expect(planning_element.errors[:project]).to eq(["can't be blank"])
+        expect(planning_element.errors[:project]).to eq(["can't be blank."])
       end
     end
 
     describe 'parent' do
-      let (:de_message) { 'darf kein Meilenstein sein' }
```

Comment:
I asked the others here after realizing that our latest move to crowdin obsoletes such specs.

The reasoning was basically: We do not have german translations in the core anymore, so we don't need to test german translations (mixed with: "WTF did anyone test this?").

So I think at least since our move to crowdin there will be no more localized specs for the core.